src/createStore.js

```diff
@@ -198,6 +199,26 @@ export default function createStore(reducer, initialState, enhancer) {
     dispatch({ type: ActionTypes.INIT })
   }
 
+  /**
+   * Interop point for observable libraries
```

Comment:
Let’s format these a little more consistent with the rest of JSDoc in this file:

* End sentences with period.
* Start parameter descriptions with a capital letter.
* I would rather avoid the awkward hanging link and instead split this over two lines, i.e. `see the observable proposal:\nhttp://...`